compute/instances/how-to/connect-to-instance-from-console.mdx

```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  title: How to connect to an Instance from the Scaleway console
+  description: This page explains how to connect to an Instance from the Scaleway console
+content:
+  h1: How to connect to an Instance from the Scaleway console
+  paragraph: This page explains how to connect to an Instance from the Scaleway console
+dates:
+  validation: 2022-02-25
+  posted: 2022-02-25
+---
+
+This page shows how to connect to your Scaleway Instance from the Scaleway console using the serial console (TTY). It provides a way to connect to your Instance for debugging or in case it lost its network connectivity. The serial console remains available even if you can not reach your Instance over the Internet.
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have [created your SSH Key and added it to your account](/console/my-project/how-to/create-ssh-key/)
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance)
+  - You have set a password for your Linux user
+
+</Message>
+
+## How to use the serial console (TTY)
```

Comment:
Shouldn't be only a section in the "Connect to an Instance" page, instead of its own page?

Reply:
actually I think you're right, that would be better?

Reply:
No. Not the same thing.

Reply:
it's still a way to connect to your Instance, so shouldn't it go in the page "Connect to your Instance"? 🤔

Reply:
I think that's what Philibeaux said also https://scaleway.slack.com/archives/C969L9C5U/p1645697310088719